tests/TestCase/Heartbeat/SensorTest.php

```diff
@@ -73,4 +73,37 @@ public function testGetStatus()
         $this->assertEquals('2017-03-30 12:45:37', $status->getLastExecuted());
         $this->assertEquals(1, $status->getSeverity());
     }
+
+    /**
+     * Tests whether the check result was fetched from cache or by running the check now
+     *
+     * @return void
+     * @covers Sensor\Status::wasCheckCached
```

Comment:
This test should be moved to StatusTest, I guess.

https://github.com/orca-services/cakephp-heartbeat/blob/9cfcd57597d9e589e0418a8e8d527263e28cf041/tests/TestCase/Heartbeat/Sensor/StatusTest.php

Reply:
Moved the test.